mlflow/store/tracking/rest_store.py

```diff
@@ -55,8 +56,13 @@ def __init__(self, get_host_creds):
         super().__init__()
         self.get_host_creds = get_host_creds
 
-    def _call_endpoint(self, api, json_body):
-        endpoint, method = _METHOD_TO_INFO[api]
+    def _call_endpoint(self, api, json_body, endpoint=None):
```

Comment:
```suggestion
    def _call_endpoint(self, api, json_body, path_params=None):
        ...
        if path_params:
            endpoint = endpoint.format(**path_params)
```

Can we extract the `getTraceInfo` path and replace `trace_id` using `path_params`?

Reply:
I feel force writing the `endpoint` is more clear because `path_params` is a free-form dict, and it relies heavily on the assumption that we are tackling endpoints like `/mlflow/api/{magic}/...`, there could be other cases we are missing.